src/postgres/src/backend/commands/tablecmds.c

```diff
@@ -796,9 +796,6 @@ DefineRelation(CreateStmt *stmt, char relkind, Oid ownerId,
 										  InvalidOid,
 										  typaddress);
 
-	/* Store inheritance information for new rel. */
```

Comment:
Why was this removed?

Reply:
I have explained this in detail in the commit message. I think this is a mistaken merge conflict artifact, as I couldn't see this line in Postgres 11.2 code.